src/cheapestTree.ts

```diff
@@ -32,3 +46,34 @@ export function cheapestTree(
   // Recalculate the correct tree price
   return calculateTreePrices(treeWithQuantityPostFlags, itemPrices)
 }
+
+function getCheaperToBuyItemIds(
+  tree: RecipeTreeWithPrices,
+  ids: Array<number> = []
+): Array<number> {
+  if (
+    typeof tree.craftDecisionPrice === 'number' &&
+    typeof tree.buyPrice === 'number' &&
+    tree.craftDecisionPrice * 0.85 > tree.buyPrice
```

Comment:
I think this is easier to understand flipped and with an explainer comment:

```ts
// If buying the item is less gold than we can get by selling the ingredients (incl. TP tax), we should just buy the item instead
tree.buyPrice < tree.craftDecisionPrice * 0.85
```

(Feel free to reword)